prod-cockpit-rcar.yaml

```diff
@@ -36,13 +36,13 @@ common_data:
       rev: beea119eb529b4a11f266004aee8b548427aea39
     - type: git
       url: "https://github.com/xen-troops/meta-xt-common.git"
-      rev: be26ff13057f8aaa503db8ef4f3826a4b483f35b
+      rev: dunfell-v0.1.0
     - type: git
       url: "https://github.com/xen-troops/meta-xt-rcar.git"
-      rev: da9485c357b041f5af6232e0a7f24bbbb3d72d56
+      rev: dunfell-v0.1.0
     - type: git
-      url: "https://github.com/xen-troops/meta-xt-prod-cockpit-rcar.git"
-      rev: master
+      url: "https://github.com/Mishytca/meta-xt-prod-cockpit-rcar.git"
```

Comment:
Please replace `Mishytca` with `xen-troops`. We should not use developer's forks in PRs.